modules/ROOT/pages/super-administrator-application-index.adoc

```diff
@@ -0,0 +1,65 @@
+= Bonita Super Administrator Application 
+:description: This page gives an easy access to the application content, and to some related topics elsewhere in the documentation.
```

Comment:
shouldn't we repeat Super Administrator application (since the description is used for SEO) ?
```suggestion
:description: This page gives an easy access to the Super Administrator application content, and to some related topics elsewhere in the documentation.
```